python-sdk/python_sdk_custom_docker_demo/main.py

```diff
@@ -0,0 +1,55 @@
+import disco
+from disco.core.constants import JobStatus
+import pathlib
+from disco.docker_image import DockerImage
+
+#we will timeout if the job ran for over 5 minutes
+#this is up to your application and change it accordingly
+LIVE_TESTS_TIMEOUT_SECONDS = 300
+
+DISCO_EMAIL = 'ray_ifx@dis.co'
+DISCO_PASSWORD = 'impact123'
```

Comment:
hardcoded credentials should be removed and replaced with a placeholder or if you check our my example, you can use environment variables instead.